tests/Support/Traits/InitCommandMockTrait.php

```diff
@@ -2,83 +2,57 @@
 
 namespace RonasIT\ProjectInitializator\Tests\Support\Traits;
 
-use Illuminate\Support\Arr;
 use RonasIT\Support\Traits\MockTrait;
 
 trait InitCommandMockTrait
 {
     use MockTrait;
 
-    public function mockFilePutContent(
-        string $exampleEnvFixtureName = 'env.example.yml',
-        string $developmentEnvFixtureName = 'env.development.yml',
-        ...$arguments,
-    ): void {
-        $callChain = [
-            ['.env.example', $this->getFixture($exampleEnvFixtureName)],
-            ['.env.development', $this->getFixture($developmentEnvFixtureName)],
-            ...$arguments,
-        ];
-
-        $this->mockNativeFunction(
-            namespace: 'RonasIT\ProjectInitializator\Commands',
-            callChain: array_map(
-                fn ($call) => $this->functionCall('file_put_contents', $call),
-                $callChain,
-            ),
-        );
+    protected function callClassExists(string $class, bool $result = true): array
+    {
+        return $this->functionCall('class_exists', [$class], $result);
     }
 
-    public function mockShellExec(array ...$rawCallChain): void
+    protected function callFileGetContent(string $fileName, string $sourceFixture): array
     {
-        $callChain = array_map(fn ($call) => $this->functionCall(
-            name: 'shell_exec',
-            arguments: Arr::wrap($call['arguments']),
-            result: Arr::get($call, 'result', 'success'),
-        ), $rawCallChain);
-
-        $this->mockNativeFunction('RonasIT\ProjectInitializator\Commands', $callChain);
+        return $this->functionCall('file_get_contents', [base_path($fileName)], $sourceFixture);
     }
 
-    public function mockFileGetContent(array ...$rawCallChain): void
+    protected function getTemplate(string $template): string
     {
-        $callChain = array_map(fn ($call) => $this->functionCall(
-            name: 'file_get_contents',
-            arguments: $call['arguments'],
-            result: $call['result'],
-        ), $rawCallChain);
+        return file_get_contents(base_path("/resources/md/readme/{$template}"));
+    }
 
-        $this->mockNativeFunction(
-            namespace: 'RonasIT\ProjectInitializator\Commands',
-            callChain: $callChain,
-        );
+    protected function getReadmePath(string $fileName): string
+    {
+        return "/vendor/ronasit/laravel-project-initializator/resources/md/readme/{$fileName}";
     }
```

Comment:
looks it should be a helper, implemented in the TestCase class and with the next logic


```suggestion
    protected function generateResourcePath(string $path): string
    {
        return "/vendor/ronasit/laravel-project-initializator/resources/{$path}";
}
```